source/driver-bulk-update.rst

```diff
@@ -13,6 +13,10 @@ Changes from previous versions
 
 Deprecated in favor of the *Driver CRUD API*.
 
+v0.8
```

Comment:
AFAIK, these version numbers have no significance but this was the closest thing to a changelog.